parquet-arrow/src/main/java/org/apache/parquet/arrow/schema/SchemaConverter.java

```diff
@@ -363,21 +357,21 @@ private TypeMapping fromParquet(Type type, String name, Repetition repetition) {
    * @return the mapping
    */
   private TypeMapping fromParquetGroup(GroupType type, String name) {
-    OriginalType ot = type.getOriginalType();
+    LogicalTypeAnnotation ot = type.getLogicalTypeAnnotation();
```

Comment:
I didn't change this because I already made a lot of "irrelevant" changes, like whitespace formatting etc. Anyway, I can rename the variable, but in that case I'd prefer a meaningful name like logicalType instead.

Reply:
I don't think this change would be irrelevant. The type has change so `ot` as OriginalType would not be meaningful.
If you change it, feel free to choose a name that is fine for you.